cypress/e2e/embedded_superset_dashboard.cy.js

```diff
@@ -0,0 +1,210 @@
+                req.reply({
+                    // Note that the string below does need to have a particular pattern: it was
+                    // copied from the network tab and cannot be replaced by a random shorter string.
+                    // The superset embed SDK must do some sort of pattern validation on it
+                    token: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7InVzZXJuYW1lIjoiYW5hbHl0aWNzIiwiZmlyc3RfbmFtZSI6IkRISVMgMiBTdXBlcnNldCIsImxhc3RfbmFtZSI6IkdhdGV3YXkifSwicmVzb3VyY2VzIjpbeyJ0eXBlIjoiZGFzaGJvYXJkIiwiaWQiOiI2NmEzNGMyYS1mYTA2LTRkYjUtYmQ2ZS0wOGZkMjRiZjVhY2MifV0sInJsc19ydWxlcyI6W10sImlhdCI6MTczNzk5NTE0My4yMTU5NjcsImV4cCI6MTczNzk5NTQ0My4yMTU5NjcsImF1ZCI6Imh0dHA6Ly8wLjAuMC4wOjgwODAvIiwidHlwZSI6Imd1ZXN0In0.AayCHirBjomllKxThOCQsn4RHoIfaXfULOVtcnylhj8',
+                })
+            } else if (req.url.includes(`${SUPERSET_BASE_URL}/embedded/`)) {
+                req.reply(SUPERSET_DASHBOARD_STUB)
+            } else {
+                // Just return the response by default
+                req.continue((resp) => resp)
+            }
+        })
+    })
+
+    it('creates an embedded superset dashboard', () => {
+        // Start a new dashboard from the start page
+        cy.visit('#/start')
+
+        cy.get(newButtonSel, EXTENDED_TIMEOUT).click()
+
+        // Choose the embedded dashboard option
+        cy.contains('External: Data from another source')
+            .should('be.visible')
+            .click()
+
+        // Click the configure source button
+        cy.contains('Configure source').should('be.visible').click()
+
+        // A modal form to create a new embedded dashboard is showing
+        cy.contains(
+            'New dashboard: configure external source (superset)'
+        ).should('be.visible')
+
+        // Check all initial values and change them
+        getInputByLabelText('Title').should('have.value', '').type(NAME)
+        getInputByLabelText('Code').should('have.value', '').type(CODE)
+        getInputByLabelText('Description', 'textarea')
+            .should('have.value', '')
+            .type(DESCRIPTION)
+        getInputByLabelText('Superset Embed ID')
+            .should('have.value', '')
+            .type(UUID)
+        getInputByLabelText('Show chart controls on dashboard items')
+            .should('be.checked')
+            .uncheck()
+        getInputByLabelText('Expand filters').should('not.be.checked').check()
+
+        // Click the create button
+        cy.contains('Save dashboard').should('be.enabled').click()
+
+        cy.contains('h3', NAME).should('be.visible')
+        cy.contains('External data').should('be.visible')
+        // An iframe should be visible with the UUID in the src
+        cy.get('iframe')
+            .should('be.visible')
+            .and('have.attr', 'src')
+            .and('contain', UUID)
+    })
+
+    it('has some options disabled in the action-bar', () => {
```

Comment:
I think this test should be part of the previous test since it's just doing more confirmation that the dashboard was created. Also, I know we're already breaking Cypress best practice by having dependencies between tests, but the less of that we have the better.